E2E-tests/config/api_config.py

```diff
@@ -116,6 +116,11 @@ class NodesApiConfig:
     selected_node: str = MISSING
     node: Node = MISSING
     token_policy_id: str = MISSING
+    permissioned_min_ariadne_d_param_value: int = MISSING
+    permissioned_max_ariadne_d_param_value: int = MISSING
+    trustless_min_ariadne_d_param_value: int = MISSING
+    trustless_max_ariadne_d_param_value: int = MISSING
+    sidechain_main_cli_network: str = MISSING
```

Comment:
I'm not sure what this means. (Is it preview-preprod etc?)
Also partner chains, not sidechain.